pkg/kubelet/cm/internal_container_lifecycle_linux.go

```diff
@@ -31,5 +34,16 @@ func (i *internalContainerLifecycleImpl) PreCreateContainer(pod *v1.Pod, contain
 		}
 	}
 
+	if i.memoryManager != nil {
+		numaNodes := i.memoryManager.GetMemoryNUMANodes(pod, container)
+		if len(numaNodes) > 0 {
```

Comment:
It's probably better to use `numaNodes.Len()` here incase the underlying abstraction ever changes.

Reply:
done